### DISCUSSION TITLE
Implementing only CPU Usage

### DISCUSSION THREAD

**Discussion by AnimeshDeb** [withcatai/node-llama-cpp#184] [General]
Hello, I am using the node js binding for a llama 2 chat model I found on the hugging face website, specifically from TheBloke. I have implemented a chat bot using the llama-2-7b-chat.Q5_K_S.gguf model from https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF . I want to ensure that only CPU is used when the AI model is running. Any advice on how to achieve this as I am relatively new to using AI models and optimizing their performance.

**Comment by giladgd** [comment]
@AnimeshDeb Using the [version 3 beta](https://github.com/withcatai/node-llama-cpp/pull/105) you can sepcify `gpu: false` when you call `getLlama({ ... })` to disable the GPU support, but I do not see any advantage to doing that if your goal is to optimize for inference performance
